storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/CloudMonitoringResult.java

```diff
@@ -147,6 +147,23 @@ public int hashCode() {
         throughput);
   }
 
+  public String generateCustomMetric() {
```

Comment:
I'd probably name this something like `formatAsCustomMetric` so it's clear this is a "taking this thing and turning it into another thing (which just happens to be a string)"